watch/watch.c

```diff
@@ -77,6 +78,7 @@ int main(int argc, char *argv[]) {
 
 			size_t point = 0;
 			for (; arg[point]; point++) {
+				/* In varietate concordia. 🇧🇷🇪🇺🤝🇺🇸🇬🇧 */
 				if (arg[point] == '.' || arg[point] == ',') {
 					arg[point] = '\0';
 					afterpoint = &arg[point + 1];
```

Comment:
Note: my interval parser has a problem somewhere i forgor good luck finding it

Reply:
It's been working fine for the last months.

Reply:
On most cases it should work fine, but when it doesn't work it's probably not easily perceptible. Still, there's an issue.

Reply:
How could we stress test this until it fails while not wasting an unacceptable time of life?